src/web/lib/storage.js

```diff
@@ -1,8 +1,12 @@
+import uuidV4 from 'uuid/v4';
+
 import { actions } from '../../lib/store';
 import { makeLog } from '../../lib/utils';
 
 const log = makeLog('web.storage');
 
+const CLIENT_UUID_KEY = 'ThemesRFun-clientUUID';
```

Comment:
Not sure if we have the final name yet, but should this be "ThemesRFun" or "Themer"?

Reply:
Doesn't really matter, as long as it's relatively unique